modules/nf-core/shinyngs/validatefomcomponents/meta.yml

```
@@ -50,15 +60,11 @@ output:
       type: file
       description: File containing validated sample metadata
       pattern: "/*.sample_metadata.tsv"
-  - sample_meta:
-      type: file
-      description: File containing validated feature metadata
-      pattern: "/*.feature_metadata.tsv"
-  - asays:
+  - assays:
```

```suggestion
  - feature_meta:
      type: file
      description: File containing validated feature metadata
      pattern: "/*.feature_metadata.tsv"
      optional: true
  - assays:
```

Sorry I obviously did a poor job of this originally!